pkg/process/config/config.go

```diff
@@ -348,14 +328,11 @@ func NewAgentConfig(loggerName config.LoggerName, yamlPath, netYamlPath string)
 		cfg.proxy = nil
 	}
 
-	// Python-style log level has WARNING vs WARN
-	if strings.ToLower(cfg.LogLevel) == "warning" {
```

Comment:
we should probably add the deprecation of the `warning` level to the release note as well.

Reply:
Done!